types/state_info.go

```diff
@@ -0,0 +1,5 @@
+package types
+
+type StateInfo struct {
```

Comment:
not sure where it's used

Reply:
i dont think its used anymore. removing it